image/templates/helm/shared/templates/02-scanner-v4-05-indexer-network-policy.yaml

```diff
@@ -63,4 +63,26 @@ spec:
   policyTypes:
   - Ingress
 {{- end }}
+{{- if ._rox.monitoring.openshift.enabled }}
```

Comment:
I think we also want to condition on `{{- if ._rox.scannerV4.exposeMonitoring }}`, right?

Reply:
This is unrelated to non-TLS monitoring, `exposeMonitoring` controls ports `:9090`.